junebug/stores.py

```diff
@@ -1,15 +1,16 @@
-from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.internet.defer import inlineCallbacks, returnValue, succeed
 from vumi.message import TransportUserMessage
 
 
 class BaseStore(object):
+    KEYSPACE = 'basestore'
```

Comment:
This should go below the docstring, otherwise the docstring is not a docstring. :)